src/env.d/03-dbus

```diff
@@ -0,0 +1,9 @@
+# Dbus settings
+DBUS_SYSTEM_BUS_ADDRESS='unix:path=/var/run/dbus/system_bus_socket'
+[[ "$DBUS_SESSION_BUS_ADDRESS"=="disabled:" ]] && unset DBUS_SESSION_BUS_ADDRESS
+if test -z "$DBUS_SESSION_BUS_ADDRESS" ; then
```

Comment:
Be consistent in style?
`if [ -z "${DBUS_SESSION_BUS_ADDRESS}" ]; then`

Reply:
Good idea.